articles/securing-your-python-application-from-hackers/index.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/securing-your-python-application-from-hackers/
+title: Securing Your Python Application from Hackers
+description: This tutorial will help the readers to secure their programs in application by going over some of the dangerous functions in Python.
+author: ahmad-mardeni
+date: 2021-05-12T00:00:00-10:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/securing-your-python-application-from-hackers/hero.png
+    alt: Securing Your Python Application from Hackers Hero image
+---
+Python has become immensely useful in cybersecurity as it supports and performs a multitude of cybersecurity functions such as malware analysis, scanning, penetration testing, etc. However, it's very easy to make a mistake in any Python code and make the program vulnerable.
+<!--more-->
+
+> It's possible to create a secure program using a programming language that is full of vulnerabilities, and it's also possible to create a vulnerable program using a programming language that's designed to be secure.
+
+In this tutorial, I'll help you avoid those mistakes which might make your program vulnerable.
+
+![We are in GIF](https://media.giphy.com/media/3oKIPcqmx1mpCOJJp6/giphy.gif)
+
+### Dangerous functions
+
+Some functions are dangerous that can be used to perform a sort of code injection or an authentication bypass. Let's put ourselves in the hacker's shoes to think like him and learn how to protect our code.
+

```

Comment:
Hi @ahmadmardeni1. Can you add the list of functions here with the links to the respective sections of the article?